hollow/src/main/java/com/netflix/hollow/core/HollowBlobOptionalPartHeader.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright 2021 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.hollow.core;
+
+import com.netflix.hollow.core.schema.HollowSchema;
+import java.util.Collections;
+import java.util.List;
+
+public class HollowBlobOptionalPartHeader {
```

Comment:
I noticed you dropped `Map<String, String>  headerTags` from the part headers? Does that mean we envision consumers to always consume the "main" part?

Reply:
That is the idea.  The main part might end up having no types (if all types were optional), but the headers would still be consumed from the main part.  This keeps the APIs consistent and backwards-compatible.